### PR TITLE
implement flask-login

### PR DESCRIPTION
**Pull request by marianodo**
To see flask-login go to URL "localhost:5000/auth/login". If you are logged in, the page will redirect to /admin/ if you are not, the page will redirect to login.html page. If you want to logout, put "localhost:5000/auth/logout" on URL.

Also, the view GET "/admin" and PUT "api/route" are protected with @login_required

### CODE REVIEW

File: app/auth/views.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
+from flask import render_template, redirect, request, url_for, flash, Flask, abort ,g
+from flask.ext.login import login_user, logout_user, login_required, current_user
+from . import auth
+from .. import db
+from .forms import LoginForm
```
Comment (avilaton) [inline app/auth/views.py]: @marianodo I get an error here, maybe you forgot to upload the `app/auth/forms.py` file?

Comment (marianodo) [inline app/auth/views.py]: I forgot delete that line.

**Comment by avilaton** [comment]
@marianodo This looks good, but let's make a few changes before we merge it. Could we simply remove the `username` field from the User model and rely simply on the email? I don't think we need the extra complication.

**Comment by marianodo** [comment]
Done. Username was remove from User model. Put Email as Username to login to the page

**Comment by avilaton** [comment]
Merged, thanks @marianodo !

**Pull request merged**